### PR TITLE
i18n(fr): Updated `authoring-content.mdx` & translated `manual-setup.md`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Updated `authoring-content.mdx` based on this commit 02c8b54a3d3d01113ad7428bf564b77f137ded1e & translated `manual-setup.md`

### CODE REVIEW

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+title: Manual Setup
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:2]:
```suggestion
title: Installation manuelle
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+description: Learn how to configure Starlight manually to add it to an existing Astro project.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:3]:
```suggestion
description: Apprendre à configurer Starlight manuellement pour l'ajouter à un projet Astro existant.
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+La façon la plus rapide de créer un nouveau site Starlight est d'utiliser `create astro` comme indiqué dans le [guide de démarrage](/fr/getting-started/#creating-a-new-project).
+Si vous voulez ajouter Starlight à un projet d'Astro existant, ce guide vous expliquera comment faire.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:9]:
```suggestion
Si vous voulez ajouter Starlight à un projet Astro existant, ce guide vous expliquera comment faire.
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+## Configurer Starlight
+
+Pour suivre ce guide, vous aurez besoin d'un projet d'Astro existant.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:13]:
```suggestion
Pour suivre ce guide, vous aurez besoin d'un projet Astro existant.
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+### Ajouter l'intégration Starlight
+
+Starlight est une [intégration d'Astro](https://docs.astro.build/fr/guides/integrations-guide/). Ajoutez-le à votre site en exécutant la commande `astro add` dans le répertoire racine de votre projet :
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:17]:
```suggestion
Starlight est une [intégration Astro](https://docs.astro.build/fr/guides/integrations-guide/). Ajoutez-la à votre site en exécutant la commande `astro add` dans le répertoire racine de votre projet :
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+    </TabItem>
+    <TabItem label="Yarn">
+        ```sh
+        yarn astro add starlight
+        ```
+    </TabItem>
+
+</Tabs>
+
+Cela installera les dépendances requises et ajoutera Starlight au tableau `integrations` dans votre fichier de configuration Astro.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:39]:
```suggestion
Cela installera les dépendances requises et ajoutera Starlight au tableau des `integrations` dans votre fichier de configuration Astro.
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+
+### Configurer l'intégration
+
+L'intégration Starlight est configurée dans votre fichier `astro.config.mjs`.
+
+Ajoutez un `title` pour commencer :
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:45]:
```suggestion
Ajoutez un titre avec la propriété `title` pour commencer :
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+
+```js {7-9}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+
+export default defineConfig({
+	integrations: [
+		starlight({
+			title: 'My delightful docs site',
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:55]:
```suggestion
			title: 'Mon ravissant site de documentation',
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+		}),
+	],
+});
+```
+
+Trouvez toutes les options disponibles dans la [référence de configuration Starlight](/fr/reference/configuration/).
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:61]:
```suggestion
Retrouvez toutes les options disponibles dans la [référence de configuration Starlight](/fr/reference/configuration/).
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+
+### Configurer les collections de contenu
+
+Starlight est construit sur le dessus des [collections de contenu d'Astro](https://docs.astro.build/fr/guides/content-collections/), qui sont configurées dans le fichier `src/content/config.ts`.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:65]:
```suggestion
Starlight s'appuie sur les [collections de contenu d'Astro](https://docs.astro.build/fr/guides/content-collections/), qui sont configurées dans le fichier `src/content/config.ts`.
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+
+Créez ou mettez à jour le fichier de configuration du contenu, en ajoutant une collection `docs` qui utilise le `docsSchema` de Starlight :
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:67]:
```suggestion
Créez ou mettez à jour le fichier de configuration du contenu, en ajoutant une collection `docs` qui utilise le schéma `docsSchema` de Starlight :
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 12]
```diff
@@ -0,0 +1,129 @@
...
+	docs: defineCollection({ schema: docsSchema() }),
+};
+```
+
+### Ajouter du contenu
+
+Starlight est maintenant configuré et il est temps d'ajouter du contenu !
+
+Créez un repetoire `src/content/docs/` et commencez par ajouter un fichier `index.md`.
+Voice la page d'accueil de votre nouveau site :
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:85]:
```suggestion
Ce sera la page d'accueil de votre nouveau site :
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 13]
```diff
@@ -0,0 +1,129 @@
...
+
+```md
+---
+# src/content/docs/index.md
+title: Mes documents
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:90]:
```suggestion
title: Ma documentation
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 14]
```diff
@@ -0,0 +1,129 @@
...
+description: En savoir plus sur mon projet dans ce site de documentation construit avec Starlight.
+---
+
+Bienvenue dans mon projet !
+```
+
+Starlight utilise le routage basé sur les fichiers, ce qui signifie que chaque fichier Markdown, MDX ou Markdoc dans `src/content/docs/` deviendra une page de votre site. Les métadonnées du frontmatter (les champs `title` et `description` dans l'exemple ci-dessus) peuvent modifier la façon dont chaque page est affichée.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:97]:
```suggestion
Starlight utilise une stratégie de routage basé sur les fichiers, ce qui signifie que chaque fichier Markdown, MDX ou Markdoc dans `src/content/docs/` deviendra une page de votre site. Les métadonnées du frontmatter (les champs `title` et `description` dans l'exemple ci-dessus) peuvent modifier la façon dont chaque page est affichée.
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 15]
```diff
@@ -0,0 +1,129 @@
...
+Voir toutes les options disponibles dans la [référence du frontmatter](/fr/reference/frontmatter/).
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:98]:
```suggestion
Retrouvez toutes les options disponibles dans la [référence du frontmatter](/fr/reference/frontmatter/).
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 16]
```diff
@@ -0,0 +1,129 @@
...
+
+## Tips pour les sites existants
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:100]:
```suggestion
## Astuce pour les sites existants
```

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 17]
```diff
@@ -0,0 +1,129 @@
...
+  - pages/
+- astro.config.mjs
+
+</FileTree>
+
+A l'avenir, nous prévoyons de mieux supporter ce cas d'utilisation pour éviter le besoin du répertoire supplémentaire dans `src/content/docs/`.
+
+### Utiliser Starlight avec SSR
+
+Maintenant, Starlight ne supporte pas [le déploiement rendu côté serveur (SSR)](https://docs.astro.build/fr/guides/server-side-rendering/) en utilisant les adaptateurs de serveur d'Astro. Nous espérons pouvoir le supporter bientôt.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/manual-setup.mdx:129]:
```suggestion
À l'heure actuelle, Starlight ne supporte pas [le déploiement avec rendu côté serveur (SSR)](https://docs.astro.build/fr/guides/server-side-rendering/) en utilisant les adaptateurs de serveur d'Astro. Nous espérons pouvoir le supporter bientôt.
```

**Review by HiDeoo — COMMENTED** [review]
Nice, let a few suggestions 👍

**Comment by dreyfus92** [comment]
Merci beaucoup pour faire la revision @HiDeoo 😁

File: docs/src/content/docs/fr/manual-setup.mdx [hunk 18]
```diff
@@ -0,0 +1,129 @@
...
+  - pages/
+- astro.config.mjs
+
+</FileTree>
+
+À l'avenir, nous prévoyons de mieux supporter ce cas d'utilisation pour éviter le besoin du répertoire supplémentaire dans `src/content/docs/`.
+
+### Utiliser Starlight avec SSR
+
+À l'heure actuelle, Starlight ne supporte pas [le déploiement rendu côté serveur (SSR)](https://docs.astro.build/fr/guides/server-side-rendering/) en utilisant les adaptateurs de serveur d'Astro. Nous espérons pouvoir le supporter bientôt.
```
Comment (delucis) [inline docs/src/content/docs/fr/manual-setup.mdx:129]: Looks like the second word in @HiDeoo’s review was missed here:

```suggestion
À l'heure actuelle, Starlight ne supporte pas [le déploiement avec rendu côté serveur (SSR)](https://docs.astro.build/fr/guides/server-side-rendering/) en utilisant les adaptateurs de serveur d'Astro. Nous espérons pouvoir le supporter bientôt.
```

**Review by delucis — COMMENTED** [review]
Thanks @dreyfus92 and for the detailed review @HiDeoo!

Noticed one thing that didn’t make it from the review comments:

File: docs/src/content/docs/fr/manual-setup.mdx (hunk 18 above)
Comment (dreyfus92) [reply docs/src/content/docs/fr/manual-setup.mdx:129]: oof, nice catch @delucis

**Comment by dreyfus92** [comment]
Thank you for the review @delucis 😁

**Pull request merged**